src/app/ecosystem-explorer/page.tsx

```diff
@@ -155,13 +164,13 @@ export default function EcosystemExplorer({ searchParams }: Props) {
                         title,
                         description,
                         image,
-                        subcategories: [categoryId],
+                        subcategory: subcategoryId,
                       } = project
 
                       const isFirstTwoImages = i < 2
                       const category = findOrThrow(
```

Comment:
I think we should rename:

```suggestion
                      const subcategory = findOrThrow(
```

I know we at some point discussed that this should be `category` but since we'll effectively still have the parent category, I think it's better we stick to `subcategory` (also in the CMS) - Does this make sense?

Reply:
As discussed, we will keep this as `category` for now.